stworzenie sekcji about

MalgorzataMarczynska opened the pull request.

ssszafir committed:
update footer-mobile

ssszafir committed:
update footer- mobile

update font-style for adress tag

DamianSikorski committed:
Gallery

Dodałem zdjęcia z usuniętym tłem, aby ustawić tło za pomocą linear-gradient.  Sekcja jest gotowa dla każdej szerokości ekranu. <picutre> które jest zakomentowane nie wyświetlało nic. ( w inspektorze pokazywała się jakaś dziwna scieżka rózniąca się od tej w vscode) Za pomocą img udało się i obrazy zmieniają się w zależności od szerokości ekranu i gęstości pikseli.

MalgorzataMarczynska committed:
stworzenie sekcji about

MalgorzataMarczynska committed:
stworzenie sekcji about

MalgorzataMarczynska committed:
stworzenie sekcji about

MalgorzataMarczynska committed:
stworzenie sekcji about

DamianSikorski committed:
gallery

MalgorzataMarczynska committed:
poprawka tła i buttonu

MalgorzataMarczynska committed:
poprawka tła i buttonu

MalgorzataMarczynska committed:
poprawka tła i buttonu

DorotaDomanska committed:
hero w wersji mobile

MalgorzataMarczynska committed:
Merge pull request #18 from KitsuneAkvma/feature/gallery

Feature/gallery

MalgorzataMarczynska committed:
Merge pull request #17 from KitsuneAkvma/feature/DorotaDomanska/hero

hero w wersji mobile

AgnieszkaBaranowska committed:
html + css products

AgnieszkaBaranowska committed:
html + css products

AgnieszkaBaranowska committed:
html + css products

kwiatkowsko committed:
03.09

MalgorzataMarczynska committed:
Merge pull request #20 from KitsuneAkvma/feature/dorota/hero

Dorota/hero

AgnieszkaBaranowska committed:
Merge branch 'feature/Agnieszka/products' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/Agnieszka/products

DamianSikorski commented on src/sass/_about.scss:
```diff
@@ -0,0 +1,102 @@
+    }
+  }
+}
+.title__label {
```
Z klasy title__label można usunać: 
 align-self: center;
  padding: 0 31px;
  margin: 0 auto;

i zastąpić:

  display: block;
  margin: 0 auto;
  max-width: 200px;

Wtedy przy szerszych telefonach ładnie to będzie zostawać na środku

DamianSikorski commented on src/sass/_about.scss:
```diff
@@ -0,0 +1,102 @@
+  padding: 0 31px;
+  margin: 0 auto;
+}
+.section__title {
```
W klasie section__title :  
  max-width: 65%;  zamienić na max-width: 200px ;
oraz dodać text-align: center ; 
Powinno to się wtedy nie rozjeżdżać i zostać ładnie na środku

MalgorzataMarczynska committed:
poprawki ułożenia tytułu i artykułu

MalgorzataMarczynska committed:
poprawki ułożenia tytułu i artykułu

MalgorzataMarczynska committed:
Merge branch 'feature/about' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/about

MalgorzataMarczynska committed:
poprawka 2 do artykulu

MalgorzataMarczynska committed:
poprawka 2 do artykulu

DamianSikorski commented:
akceptacja

DamianSikorski reviewed (commented):
Akceptacja

DamianSikorski committed:
wtępne ukończenie skecji advantages

DamianSikorski committed:
wtępne ukończenie skecji advantages

DamianSikorski committed:
wtępne ukończenie skecji advantages

KitsuneAkvma committed:
Reviews/mobile

Skończona sekcja ( bez funkcjonującego slidera), tylko wersja mobile.

KitsuneAkvma committed:
feature/footer poprawka/implementacja zmiennych

Mały fix ze względu na to, że nie ma dzisiaj Kamila

KitsuneAkvma committed:
Merge branch 'feature/advantages' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/advantages

KitsuneAkvma committed:
Advantages with rebase

Damianowi nie działało  to ja wrzucam

kwiatkowsko committed:
Update main.scss

kwiatkowsko committed:
Merge pull request #7 from KitsuneAkvma/feature/footer-mobile

add footer-mobile

DamianSikorski committed:
mała poprawka

DamianSikorski committed:
Merge branch 'feature/advantages' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/advantages

KitsuneAkvma committed:
feature/reviews aktualizacja zmiennych

KitsuneAkvma committed:
Merge branch 'dev' into feature/advantages

kwiatkowsko committed:
Merge pull request #24 from KitsuneAkvma/feature/advantages

add feature advantages-mobile

KitsuneAkvma committed:
feature.reviews/skrócenie kodu

KitsuneAkvma committed:
feature/footer/fix zaktualizowano zmienne

KitsuneAkvma committed:
update footer variables

KitsuneAkvma committed:
MAJOR FIX FOOTER & REVIEW

!!!!!!!!!!!!!!!!!!!!!

kwiatkowsko committed:
Merge pull request #25 from KitsuneAkvma/feature/footer

Feature/footer/patch

kwiatkowsko committed:
publish scss and html contacts

kwiatkowsko committed:
Merge pull request #22 from KitsuneAkvma/feature/reviews

Reviews/mobile fixed

kwiatkowsko committed:
buttons changes

KitsuneAkvma committed:
Merge pull request #26 from KitsuneAkvma/feature/agata/contacts_2

Feature/agata/contacts

KitsuneAkvma committed:
features/reviews/fix

Fixed broken layout ( font size in rood has been changed and it broke whole structure)

kwiatkowsko committed:
Merge pull request #27 from KitsuneAkvma/feature/reviews

pięknie

AgnieszkaBaranowska committed:
update products mobile

AgnieszkaBaranowska committed:
update products mobile

AgnieszkaBaranowska committed:
Merge branch 'feature/Agnieszka/products' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/Agnieszka/products

kwiatkowsko committed:
Merge pull request #21 from KitsuneAkvma/feature/Agnieszka/products

Feature/agnieszka/products

MalgorzataMarczynska committed:
Merge branch 'feature/about' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/about

MalgorzataMarczynska committed:
poprawa ułożenia article

MalgorzataMarczynska committed:
poprawka buttona2

MalgorzataMarczynska committed:
Merge branch 'feature/about' of https://github.com/KitsuneAkvma/tankteam-project-1 into feature/about
